api/shape/Plan.js

```diff
@@ -46,12 +46,12 @@ export const computeSides = (c1, c2, sides, zag = 0.01) => {
 export const zagSides = Shape.registerMethod2(
   'zagSides',
   ['number', 'number'],
-  (diameter = 1, zag = 0.01) => 1 / toSidesFromZag(diameter, zag)
+  (diameter = 1, zag = 0.01) => toSidesFromZag(diameter, zag)
```

Comment:
What does this change mean? Why was it 1/ before...ah was the number of segments?

Reply:
I managed to confuse it with zagStep :)